File: Cpp_Course/Cpp_Course/Files/getting_user_input.c
#include <iostream>

using namespace std;

int main()
{
    int age;
    cout << "Enter your age: " << endl;
    cin >> age;
    cout << "You are " << age << " years old." << endl;

    cout << "\n" << endl;

    double dage;
    cout << "Enter your age: " << endl;
    cin >> dage;
    cout << "You are " << dage << " years old." << endl;


    cout << "\n" << endl;

    char cage;
    cout << "Enter your age: " << endl;
    cin >> cage;
    cout << "You are " << cage << " years old." << endl;

    cout << "\n" << endl;

    // Strings are different

    string name;
    cout << "Enter your name: ";
    getline(cin, name);

    cout << "Hello " << name << "." << endl;

    return 0;
}

File: Ruby_Course/Basics/getting_user_input.rb
puts "Hello"

puts "Enter your name: "
name = gets
puts ("Hello " + name)

puts ("Hello " + name + ", you are amazing!")

# Pritns everything after name into a new line. In oder to avoid this, use .chomp()
puts "Enter your name: "
name = gets.chomp()
puts ("Hello " + name + ", you are amazing!")
puts "Enter your age: "
age = gets.chomp()
puts ("Hello " + name + ", you are " + age + "!")
